### DISCUSSION TITLE
Preact refresh CLS issues with Tailwind

### DISCUSSION THREAD

**Discussion by Caligulaa** [preactjs/preact-cli#1719] [General]
I'm trying to use tailwind with preact. Everything works well in dev.

In prod my breakpoint size classes cause the layout to flash on first page load/refresh.
Looks like anything in "lg:mt-2....." takes a few ms to be applied, causing weird CLS.

Anyone know how to fix this behavior?

Thanks <3

**Comment by rschristian** [comment]
Is this with Preact-CLI?

**Reply by Caligulaa** [reply]
Yes, visible on resistance.biz the main header text content flash the small screen version first, then display what ever is in my lg: parameters.

**Reply by rschristian** [reply]
Please make an issue in that case with a minimum reproduction.

**Reply by Sirleid** [reply]
Sim
